torch/csrc/jit/runtime/register_distributed_ops.cpp

```diff
@@ -26,7 +26,7 @@ static auto workerInfo =
 
 RegisterOperators reg_rpc_ops(
     {Operator(
-         "aten::to_here(RRef(t) self) -> t",
+         "aten::to_here(RRef(t) self) -> t(*)",
```

Comment:
Curious, do we also change `t` to `t(*)` in `RRef(t)`?

Reply:
not needed, the change to the return type is to ensure the return value is aliased to any other values so that the dce will not remove this op, the `RRef(t)` does not need to alias to any other value like the return value